packages/unhead/src/createHead.ts

```diff
@@ -25,9 +25,9 @@ import { IsBrowser } from './env'
 
 export const CorePlugins = () => [
   // dedupe needs to come first
+  TemplateParamsPlugin(),
```

Comment:
Should be safe to revert this, the deduping was fixed with https://github.com/unjs/unhead/commit/f9bd644d125d7319298ee8028e5f9936c11f9d71